---
layout: default
title: Softwares
subtitle: Developed Softwares
use-site-title: true
---

<div class="container">
    <div class="row">
        <h2>Softwares</h2>
    <div class="row">
        <h3>DISPbind: disorder protein genomic binding analysis toolkit for DisP-seq<br></h3>
				<div class="col-lg-3">
          <img src="/png/softwares/DISPbind.jpg" width="300" height="200">
        </div>
        <div class="col-lg-9">
            <p>
						DISPbind is designed for DisP-seq datasets, providing essential functionalities such as genome mapping, bigwig generation and DisP islands identification.<br>
            Github: <a href="https://github.com/rdong08/DISPbind">https://github.com/rdong08/DISPbind</a><br>
            Citation: Xing YH#, <b>Dong R#</b>, Lee L, Rengarajan S, Riggi N, Boulay G, Rivera N. M*. DisP-seq reveals the genome-wide functional organization of DNA-associated disordered proteins. <a href="https://www.nature.com/articles/s41587-023-01737-4">Nat. Biotechnol., 2023: 1-13.</a><br>
            </p>
        </div>
    </div>
    <div class="row">
      <h3>SpatialDWLS: accurate deconvolution of spatial transcriptomic data<br></h3>
				<div class="col-lg-3">
          <img src="/png/softwares/SpatialDWLS.jpg" width="300" height="200">
        </div>
        <div class="col-lg-9">
            <p>
            SpatialDWLS is an accurate and computationally efficient method for estimating the spatial distribution of cell types from spatial transcriptomic data. It provides a valuable enabling toolkit for investigating cell-cell interactions from various spatial transcriptomic technology platforms that do not have single-cell resolution.<br>
            Github: <a href="https://github.com/rdong08/spatialDWLS_dataset">SpatialDWLS example dataset.</a> The SpatialDWLS has beed implemented into <a href="https://github.com/drieslab/Giotto">Giotto</a>.<br>
            Citation: <b>Dong R</b>, Yuan GC*, SpatialDWLS: accurate deconvolution of spatial transcriptomic data. <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02362-7">Genome Biol., 2021, 22(1): 1-10.</a><br>
            </p>
        </div>
    </div>
    <div class="row">
        <h3>GiniClust3: rare cell identification from large scale single cell RNA-seq datasets<br></h3>
				<div class="col-lg-3">
            <img src="/png/softwares/GiniClust3.jpg" width="300" height="200">
        </div>
        <div class="col-lg-9">
            <p>
						GiniClust3 is specifically tailored for large-scale scRNA-seq dataset for rare cell type identification.<br>
            Github: <a href="https://github.com/rdong08/GiniClust3">https://github.com/rdong08/GiniClust3</a><br>
            Citation: <b>Dong R</b>, Yuan GC*, GiniClust3: a fast and memory-efficient tool for rare cell type identification. <a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-020-3482-1">BMC Bioinform., 2020 21: 1-7.</a><br>
            </p>
        </div>
    </div>
    </div>
</div>
