---
layout: default
title: Rui Dong's CV
permalink: /cv/
---

<div style="text-align:center;"># Rui Dong</div>
Research fellow in Department of Pathology, Massachusetts General Hospital, Harvard Medical School.

<div id="webaddress">
<a href="rdong@mgh.harvard.edu">rdong@mgh.harvard.edu</a>
</div>


### Specialized in

Computational Biology, Transcriptomics, Epigenomics


## Education

`2012/09-2017/07`
__CAS-MPG Partner Institute for Computational Biology, SIBS, CAS, China.__

- **Ph.D.** with Prof. **Li Yang**
- Major: Computational biology

`2007/09-2011/07`
__Shandong University, Shandong, China.__

- **B.S.** Major: Biological sciences



## Awards

`2017` Shanghai Outstanding Graduate Award

`2016` National Scholarship for Outstanding Graduate Student, Ministry of Education of the People's Republic of China

`2016` Zhu Li Yuehua Outstanding Doctoral Scholarship, Chinese Academy of Sciences

`2015` Merit Student, University of Chinese Academy of Sciences


## Publications

<!-- A list is also available [online](https://scholar.google.com/citations?user=wcQO0GAAAAAJ&hl=zh-CN) -->
You can visit [Google Scholar](https://scholar.google.com/citations?user=wcQO0GAAAAAJ&hl) for more details<br>

\# Co-first authors; \* Corresponding authors

### **2023**

* Xing YH\#, **Dong R**\#, Lee L, Rengarajan S, Riggi N, Boulay G, Rivera N. M\*. DisP-seq reveals the genome-wide functional organization of DNA-associated disordered proteins. ***Nat. Biotechnol.***, 2023: 1-13.
* Sanalkumar R\#, **Dong R**\#, Xing YH, Lee L, Iyer S, Letovanec I, Rosa S, Chebib I, Nielsen P, Renella R, Cote G, Choy E, Aryee M, Stegmayer K, Stamenkovic I, Rivera N. M\*, Riggi N\*. Highly connected 3D chromatin networks established by an oncogenic fusion protein shape tumor cell identity. ***Sci. Adv.***, 2023, 9(13): eabo3789.

### **2022**

* Yasuhara T, Xing YH, Bauer N, Lee L, **Dong R**, Yadav T, Soberman R, Rivera M, Zou L, Condensates induced by transcription inhibition localize active chromatin to nucleoli. ***Mol. Cell***, 2022, 82(15): 2738-2753.
* MÃ¶ller E, Praz V, Rajendran S, **Dong R**, Cauderay A, Xing YH, Lee L, Fusco C, Broye L, Cironi L, Iyer S, Rengarajan S, Awad M, Naigles B, Letovanec I, Ormas N, Finzi G, Rosa S, Sessa F, Chebib I, Nielsen G, Digklia A, Spentzos D, Cote G, Choy E, Aryee M, Stamenkovic I, Boulay G, Rivera M, Riggi N. EWSR1-ATF1 dependent 3D connectivity regulates oncogenic and differentiation programs in Clear Cell Sarcoma. ***Nat. Commun.***, 2022, 13(1): 1-18.

### **2021**
                                                                                                                                                                                                                                                                                                                          
* **Dong R**, Yuan GC\*, SpatialDWLS: accurate deconvolution of spatial transcriptomic data. ***Genome Biol.***, 2021, 22(1): 1-10.
* Dries R\#, Zhu Q\#, **Dong R**, Eng CH, Li H, Liu K, Fu Y, Zhao T, Sarkar A, Bao F, George R, Pierson N, Cai L, Yuan GC\*. Giotto, a toolbox for integrative analysis and visualization of spatial expression data. ***Genome Biol.***, 2021, 22(1): 1-31.

### **2020**

* **Dong R**, Yuan GC\*, GiniClust3: a fast and memory-efficient tool for rare cell type identification. ***BMC Bioinform.***, 2020 21: 1-7.

### **2019**

*	**Dong R**, Ma XK, Chen LL and Yang L\*. Genome-Wide Annotation of circRNAs and Their Alternative Back-Splicing/Splicing with CIRCexplorer Pipeline. ***Methods Mol. Biol.***, 2019. vol 1870. 
* Ma XK, Wang MR, Liu CX, **Dong R**, Carmichael GG, Chen LL and Yang L\*. CIRCexplorer3: A CLEAR pipeline for direct comparison of circular and linear RNA expression. ***Genomics Proteomics Bioinformatics***, 2019, 17(5): 511-521.
* Akerberg B\#, Gu F\#, Dusen N, Zhang X, **Dong R**, Li K, Zhang B, Zhou B, Sethi I, Ma Q, Wasson L, Wen T, Liu J, Dong K, Conlon F, Zhou J, Yuan GC, Zhou P, Pu W\*. A reference map of murine cardiac transcription factor chromatin occupancy identifies dynamic and conserved enhancers. ***Nat. Commun.***, 2019, 10(1): 1-16.
* Tsoucas D\*, **Dong R**, Chen H, Zhu Q, Guo G, Yuan GC\*. Accurate estimation of cell-type composition from gene expression data. ***Nat. Commun.***, 2019, 10(1): 2975.

### **2018**

* **Dong R**\#, Ma XK\#, Li GW, Yang L\*. CIRCpedia v2: An Updated Database for Comprehensive CircRNA Annotation and Expression Comparison. ***Genomics Proteomics Bioinformatics***, 2018, 16(4), 226-233.

### **Before 2018**

* **Dong R**\#, Ma XK\#, Chen LL\*, Yang L\*. Increased complexity of circRNA expression during species evolution. ***RNA Biol.***, 2017, 14(8):1064-1074.
* Xing YH\#, Yao RW\#, Zhang Y\#, Guo CJ, Jiang S, Xu G, **Dong R**, Yang L and Chen LL\*. SLERT Regulates DDX21 Rings Associated with Pol I Transcription. ***Cell***, 2017, 169(4): 664-678.
* Zhong C\#, Xie Z\#, Yin Q\#, **Dong R**, Yang S, Wu Y, Yang L and Li J\*. Parthenogenetic haploid embryonic stem cells efficiently support mouse generation by oocyte injection. ***Cell Res.***, 2016, 26(1):131-4.
* Zhang XO\#, **Dong R**\#, Zhang Y\#, Zhang JL, Luo Z, Zhang J, Chen LL\* and Yang L\*. Diverse alternative back-splicing and alternative splicing landscape of circular RNAs. ***Genome Res.***, 2016, 26(9):1277-87.
* **Dong R**, Zhang XO, Zhang Y, Ma XK, Chen LL and Yang L\*. CircRNA-derived pseudogenes. ***Cell Res.***, 2016, 26(6):747-50.
* Zhong C\#, Yin Q\#, Xie Z\#, Bai M\#, **Dong R**\#, Tang W, Xing YH, Zhang H, Yang S, Chen LL, Bartolomei MS, Ferguson-Smith A, Li D, Yang L\*, Wu Y\* and Li J\*. CRISPR-Cas9-Mediated Genetic Screening in Mice with Haploid Embryonic Stem Cells Carrying a Guide RNA Library. ***Cell Stem Cell***, 2015, 17(2):221-32.
* Chen T\#, Xiang JF\#, Zhu S\#, Chen S, Yin QF, Zhang XO, Zhang J, Feng H, **Dong R**, Li XJ, Yang L\* and Chen LL\*. ADAR1 is required for differentiation and neural induction by regulating microRNA processing in a catalytically independent manner. ***Cell Res.***, 2015, 25(4):459-76.

<!-- ### Footer

Last updated: Aug 24 2023 -->
